app/models/manageiq/providers/kubevirt/inventory/parser.rb

```diff
@@ -111,6 +111,8 @@ def process_node(object)
       :host    => host_object,
       :storage => storage_object,
     )
+
+    host_storage_object
```

Comment:
I think this isn't correct. Why should the `process_node` method return the `host_storage_object`? If Rubocop complains about that I'd suggest to just return `nil` explicitly.

Reply:
what I should have done is just removing the assignment to `host_storage_object`

Reply:
Yes. You may still want to return `nil` explicitly, because otherwise the method will return the same object anyhow. Not really important, as nothing uses the value returned by this method.